frontend/src/pages/about/index.js

```diff
@@ -7,17 +7,20 @@ import Strategy from "../../components/page-about/index/strategy"
 import Belief from "../../components/page-about/index/belief"
 import LeadershipSection from "../../components/page-about/index/leadership";
 import Feedback from "../../components/page-about/index/feedback"
+import Story from "../../components/page-about/index/story"
```

Comment:
Did you forget to add this file?

Reply:
Oh yea, I will take that out from index.js because it's not completed yet. I just added it to see it on the website as I work.